extend.sh

```diff
@@ -182,6 +219,24 @@ function resolve_device_name(){
     fi
 }
 
+
+function deactivate_volume_group(){
+    volume_group_name=$(pvs "$DEVICE_NAME""$SUCCESSIVE_PARTITION_NUMBER" -o vg_name --noheadings|/usr/bin/sed 's/^[[:space:]]*//g')
```

Comment:
Should these variables be local as well?

Reply:
Yeah, I prefer to keep them local to avoid conflicts.

Reply:
Done